src/formats/XTC.cpp

```diff
@@ -85,3 +121,17 @@ void XTCFormat::set_cell(matrix box, Frame& frame) {
     // Factor 10 because the cell lengthes are in nm in the XTC format
     frame.set_cell({a.norm() * 10, b.norm() * 10, c.norm() * 10, alpha, beta, gamma});
 }
+
+void XTCFormat::get_cell(matrix box, const Frame& frame) {
+    // Factor 10 because the cell lengthes are in nm in the XTC format
+    const auto& matrix = frame.cell().matrix() / 10.0;
```

Comment:
`const auto&` can be replaced by `auto` here: you are creating a new matrix in the division anyway.

Reply:
Done.